felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -343,6 +341,45 @@ type bpfEndpointManager struct {
 	natOutIdx int
 
 	arpMap maps.Map
+
+	v4 *bpfEndpointManagerDataplane
+	v6 *bpfEndpointManagerDataplane
+}
+
+type bpfEndpointManagerDataplane struct {
```

Comment:
Perhaps better to think of this like `bpfEpIPFamilyMgr` which manages what is specific for the ip family and it uses the `bpfDataplane` to achieve it?